.gometalinter.json

```diff
@@ -1,5 +1,7 @@
 {
   "Exclude": [
-    "exported (\\w+) ([\\w\\.]+) should have comment or be unexported"
+    "exported (\\w+) ([\\w\\.]+) should have comment or be unexported",
+    "templates/templates-packr.go",
+    "error return value not checked \\(defer"
```

Comment:
i don't really know this dsl, but seems like `\\(defer` might be incomplete/typo